packages/client/components/TeamPrompt/Recurrence/UpdateRecurrenceSettingsModal.tsx

```diff
@@ -43,21 +43,36 @@ const ActionsContainer = styled('div')({
   padding: 16
 })
 
-const UpdateButton = styled(PlainButton)({
+const ActionButton = styled(PlainButton)({
   height: 36,
-  backgroundColor: PALETTE.SKY_500,
-  color: PALETTE.WHITE,
   padding: '0px 16px',
   textAlign: 'center',
-  borderRadius: 32,
-  cursor: 'pointer',
+  borderRadius: 32
+})
+
+const UpdateButton = styled(ActionButton)<{disabled: boolean}>(({disabled}) => ({
+  backgroundColor: PALETTE.SKY_500,
+  color: PALETTE.WHITE,
   '&:hover': {
-    backgroundColor: PALETTE.SKY_600
+    backgroundColor: disabled ? undefined : PALETTE.SKY_600
```

Comment:
+1 there should be no need to do that, `PlainButton` has opacity configured when disabled, see [here](https://github.com/ParabolInc/parabol/blob/6ab16235ef6abcbd1c4ae69662974a031ae0ea0e/packages/client/components/PlainButton/PlainButton.tsx#L38)

Reply:
Eh it's a little different, but the default `PlainButton` style is also reasonable. Changed.